### ISSUE TITLE
multiple occurrence of properties in xml template definition

### ISSUE THREAD

**Issue by jbfewo** [sulu/sulu#3046]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | maybe
| Sulu Version | 11c56743dca75d8d91c6c6b0ef67c8f0f56501a6
| Browser Version | -

#### Actual Behavior

The template-1.0.xsd says it is possible to set a min/maxOccurs-attribute to a property but doing so  have no impact. https://github.com/sulu/sulu/blob/bce4655162e213c791d34c2fe82f8009a0623494/src/Sulu/Component/Content/Metadata/Loader/schema/template-1.0.xsd#L105-L106

#### Expected Behavior

It should be possible to add the specific element multiple times.

#### Steps to Reproduce

```
<type name="navigation_box">
    <meta>
        <title lang="en">Navigation box</title>
    </meta>
    <properties>
        <property name="title" type="text_line" mandatory="true">
            <meta>
                <title lang="en">Title</title>
            </meta>
        </property>
        <property name="list_item" type="text_line" mandatory="true" minOccurs="1" maxOccurs="10">
            <meta>
                <title lang="en">List element</title>
            </meta>
        </property>
    </properties>
</type>
```

#### Possible Solutions

Change the xsd to remove the expectation or implement a new feature.

**Comment by danrot** [comment]
@wachterjohannes Hasn't that already been working? Can confirm that it doesn't work at the moment...

**Comment by alexander-schranz** [comment]
@danrot I did tried it some time ago (1.2, 1.3) and it did work the UI did look a little deprecated for it but it did work.

**Comment by wachterjohannes** [comment]
@danrot it has worked but i cannot say since when it is broken ...

**Comment by danrot** [comment]
@wachterjohannes Maybe that's something you can try to fix?

**Comment by wachterjohannes** [comment]
i could try (:

**Comment by danrot** [comment]
Now I remember... if you set `minOccurs` also to something higher than 1 it will work. That's an issue @akira28 already tried to solve at https://github.com/sulu/sulu/pull/2653. But I am not happy with the solution there...

**Comment by danrot** [comment]
Fixed in #3108

**Comment by danrot** [comment]
However, two other issues arouse (have probably also already been here before: #3115 and #3116)
